platforms/android/android.toolchain.cmake

```diff
@@ -1330,6 +1350,10 @@ if( ARMEABI_V7A )
  set( ANDROID_LINKER_FLAGS "${ANDROID_LINKER_FLAGS} -Wl,--fix-cortex-a8" )
 endif()
 
+if( ARMEABI_V7A_HARD )
+ set( ANDROID_LINKER_FLAGS "${ANDROID_LINKER_FLAGS} -Wl,--no-warn-mismatch -lm_hard" )
```

Comment:
Updated with those two changes. (remove duplicate linker flags and add -arm for debug)